src/services/copyConfig/recaptcha/recaptchaConfiguration.js

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright: Copyright 2023 SAP SE or an SAP affiliate company and cdc-tools-chrome-extension contributors
+ * License: Apache-2.0
+ */ 
+
+import Recaptcha from '../../recaptcha/recaptcha.js'
+import Policy from '../policies/policies.js'
+import RiskProviders from '../../recaptcha/riskProviders.js'
+
+class RecaptchaConfiguration {
+  #credentials
+  #site
+  #dataCenter
+  #recaptcha
+  #policy
+  #riskProviders
+
+  constructor(credentials, site, dataCenter) {
+    this.#credentials = credentials
+    this.#site = site
+    this.#dataCenter = dataCenter
+    this.#recaptcha = new Recaptcha(credentials.userKey, credentials.secret, credentials.gigyaConsole)
+    this.#policy = new Policy(credentials, site, dataCenter)
+    this.#riskProviders = new RiskProviders(credentials.userKey, credentials.secret, credentials.gigyaConsole)
+  }
+
+  async get() {
+    try {
+      const recaptchaResponse = await this.getRecaptcha().get(this.#site, this.#dataCenter)
+      if (recaptchaResponse.errorCode !== 0) {
+        throw new Error(`Error fetching reCAPTCHA policies: ${recaptchaResponse.errorMessage}`)
+      }
+
+      const policiesResponse = await this.#policy.get()
+      if (policiesResponse.errorCode !== 0) {
+        throw new Error(`Error fetching security and registration policies: ${policiesResponse.errorMessage}`)
+      }
+
+      const riskProvidersResponse = await this.#riskProviders.get(this.#site, this.#dataCenter)
+      if (riskProvidersResponse.errorCode !== 0) {
+        throw new Error(`Error fetching Risk Providers configuration: ${riskProvidersResponse.errorMessage}`)
+      }
+
+      return {
+        errorCode: recaptchaResponse.errorCode,
+        recaptchaConfig: recaptchaResponse.Config,
+        securityPolicies: policiesResponse.security,
+        registrationPolicies: policiesResponse.registration,
+        riskProvidersConfig: riskProvidersResponse.config,
+      }
+    } catch (error) {
+      console.error(`Error in RecaptchaConfiguration.get: ${error}`)
```

Comment:
I'm having this error with this URL in localhost: http://localhost:3000/#/99999999/4_TU0stSDgnidpm7kYW0-Wfg/cdc-toolbox/copy-configuration-extended


<img width="1713" alt="image" src="https://github.com/user-attachments/assets/98f4a802-98c7-4ccb-8dce-03b7445c5eff">

Reply:
The error message is empty:
<img width="890" alt="image" src="https://github.com/user-attachments/assets/5008b25d-7803-4c23-a4ca-a039f82e9a56">